lib/rules/complexity.js

```diff
@@ -12,7 +12,7 @@
 
 module.exports = function(context) {
 
-    var THRESHOLD = context.options[0];
+    var THRESHOLD = (typeof context.options[0] !== "undefined") ? context.options[0] : 20;
```

Comment:
Can't do a simple `context.options[0] || 20` here, because it would change an explicit `0` into `20`.